ironpython-git/PKGBUILD

```diff
@@ -1,19 +1,17 @@
 # Maintainer: Michael Schubert <mschu.dev at gmail>
 
 pkgname=ironpython-git
-pkgver=2.7.2.a2.471.g7be8b73
+pkgver=ac72964d7
 pkgrel=1
 pkgdesc="Python implementation for the .NET framework"
 arch=("any")
 url="http://ironpython.net"
 license=("Apache")
-depends=('mono')
+depends=('mono' 'msbuild-bin')
```

Comment:
This should be in `makedepends`, not `depends`, if it's just used to build the software but not required to run it.